### PR TITLE
BOAC-1330: displays searched student at top of class list

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-1330

This satisfies the first requirement; will amend or submit a follow-up for the second.

### CODE REVIEW

File: boac/static/app/course/courseController.js [hunk 1]
```diff
@@ -78,9 +78,18 @@
       }
     });
 
+    var processResponse = function(data) {
```
Comment (johncrossman) [inline boac/static/app/course/courseController.js]: All looks good, @lyttam  My only request: a different name for `processResponse` function. Something that is self-documenting?

Comment (lyttam) [reply boac/static/app/course/courseController.js]: @johncrossman I was on the fence about the name. I can change it to something more specific like `reorderStudents` or `featureSearchedStudent`

Comment (johncrossman) [reply boac/static/app/course/courseController.js]: +1 on `featureSearchedStudent`

File: boac/static/app/course/courseController.js [hunk 2]
```diff
@@ -78,9 +78,18 @@
...
+        return student.uid === _.get($location.search(), 'u');
+      });
+      section.students = _.union(subject, section.students);
+      return section;
+    };
+
     var updateCourseData = function(response) {
       $rootScope.pageTitle = response.data.displayName;
-      $scope.section = response.data;
+      $scope.section = featureSearchedStudent(response.data);
```
Comment (lyttam) [inline boac/static/app/course/courseController.js]: Amended, @johncrossman.

**Comment by johncrossman** [comment]
Travis failure is due to an updated "flake8" dependency –– Python linting is suddenly stricter.  The offending py code can be fixed in a separate PR.

**Pull request merged**